src/main/scala/org/ergoplatform/http/api/ApiCodecs.scala

```diff
@@ -52,14 +51,12 @@ trait ApiCodecs extends JsonCodecs {
 
   implicit val proveDlogEncoder: Encoder[ProveDlog] = _.pkBytes.asJson
 
-  implicit val encodedTokenIdEncoder: Encoder[EncodedTokenId] = _.asJson
-
   implicit val balancesSnapshotEncoder: Encoder[WalletDigest] = { v =>
     import v._
     Json.obj(
       "height" -> height.asJson,
       "balance" -> walletBalance.asJson,
-      "assets" -> walletAssetBalances.map(x => (x._1: String, x._2)).asJson
+      "assets" -> walletAssetBalances.toMap.map(x => (x._1: String, x._2)).asJson
```

Comment:
Does this toMap conversion necessary?

Reply:
Yes, output JSONs are different, so to avoid breaking backwards compatibility .toMap is needed

Reply:
Please add this comment to the code.

Reply:
done